staging/src/k8s.io/client-go/util/flowcontrol/backoff.go

```diff
@@ -136,9 +151,26 @@ func (p *Backoff) DeleteEntry(id string) {
 	delete(p.perItemBackoff, id)
 }
 
+// StartBackoffGC used to start the Backoff garbage collection mechanism
+// Backoff.GC() will be executed each minute
+func StartBackoffGC(backoff *Backoff, stopCh <-chan struct{}) {
```

Comment:
I'm trying to see who calls `StartBackoffGC()`, but I can only find evidence of it in the federation code.

Reply:
It will be used in the PR: https://github.com/kubernetes/kubernetes/pull/51153

It is the same code that in the ```federation.JobController``` package, but move to a more common package.